zubhub_backend/zubhub/creators/serializers.py

```diff
@@ -199,6 +198,9 @@ class CustomRegisterSerializer(RegisterSerializer):
                                             queryset=Location.objects.all())
     bio = serializers.CharField(allow_blank=True, default="", max_length=255)
     subscribe = serializers.BooleanField(default=False)
+    creator_tags = serializers.SlugRelatedField(slug_field='name',
+                                                queryset=CreatorTag.objects.all(),
+                                                many=True)
```

Comment:
we'd need to adjust this because we expect just a single tag string (again if you think there is a reason we need to expect an array please I'd like to hear)

Reply:
moved to a string, the Array was in case we wanted to create a user with several roles, but it is okay we can do that from the Django admin panel